cmd/compose/compose.go

```diff
@@ -169,7 +169,12 @@ func (o *projectOptions) toProject(services []string, po ...cli.ProjectOptionsFn
 
 	ef := o.EnvFile
 	if ef != "" && !filepath.IsAbs(ef) {
-		ef = filepath.Join(project.WorkingDir, o.EnvFile)
```

Comment:
I'm not sure I understand the intent here. afaict `filepath.Join(project.WorkingDir, o.EnvFile)` should already make this path absolute, but maybe I missed something?
could you please clarify this in your PR description as the Docker Desktop issue is not public

Reply:
In the example where:
- `/some/path` is where we execute the command
- `/some/path/wrkdir` is the path passed as `--project-directory`
- `./wrkdir/envs/my.env` as the path passed in `--env-file`

in this we would end with `/some/path/wrkdir` + `./wrkdir/envs/my.env` which would repeat `wrkdir` resulting in `/some/path/wrkdir/wrkdir/envs/my.env` which is wrong.

Reply:
ah ok, because we resolve envfile based on project dir (which is definitely a bug) and we should resolve relative to command working dir. Then `filepath.Abs` is relevant. Earlier version of the diff I reviewed was doing some voodoo magic with `WorkingDir` so my confusion